daemon/api.go

```diff
@@ -2519,6 +2519,17 @@ func postDebug(c *Command, r *http.Request, user *auth.UserState) Response {
 		}, nil)
 	case "can-manage-refreshes":
 		return SyncResponse(devicestate.CanManageRefreshes(st), nil)
+	case "connectivity":
+		// FIXME: add to snapstate.StoreService? feels not ideal :/
+		//        other options?
+		s := snapstate.Store(st).(*store.Store)
+		st.Unlock()
+		failed, err := s.ConnectivityCheck()
+		st.Lock()
+		if err != nil {
+			return InternalError("cannot run connectivity check: %v", err)
+		}
+		return SyncResponse(failed, nil)
```

Comment:
mmh, I think having simply "{}"  as the happy response is a bit confusing, also having the domains as top level makes it hard to evolve this if needed

Reply:
I updated it to return something more rich, please let me know if that looks better.